### DISCUSSION TITLE
Using Mojoc code in an existing game engine.

### DISCUSSION THREAD

**Discussion by dxur** [scottcgi/Mojoc#23] [General]
I'm making the laMaEngine a c99 ECS game engine and I'm impressed of the similarities between my project and yours, My engine now have a PBR vulkan rendering with ray/path tracing supported, I want to use parts from your engine in my engine so if that possible since the engine is under MIT license.

**Comment by scottcgi** [comment]
Sure and Welcome, that's the meaning of MIT open source.
